examples/tensorflow/text-classification/run_glue.py

```diff
@@ -368,6 +352,10 @@ def main():
                 f"model labels: {list(sorted(label_name_to_id.keys()))}, dataset labels: {list(sorted(label_list))}."
                 "\nIgnoring the model labels as a result.",
             )
+            label_to_id = {label: i for i, label in enumerate(label_list)}
+    if label_to_id is not None:
+        config.label2id = label_to_id
+    config.id2label = {id: label for label, id in config.label2id.items()}
```

Comment:
```suggestion
        config.id2label = {id: label for label, id in config.label2id.items()}
```
No point doing that otherwise I think.

Reply:
Fastest reviewing ever!